iina/ViewLayer.swift

```diff
@@ -347,6 +347,40 @@ class ViewLayer: CAOpenGLLayer {
     return ctx
   }
 
+  // MARK: - ICC Profile
```

Comment:
Why was this put in `ViewLayer`? Looks like it would fit better in `MPVController`. You could pull out the `videoView.$isUninited.withLock` into the calling method to keep it inside `VideoView` and reduce level of nested code.

Reply:
Sorry I'm slow in responding. Been focused on the JPEG XL problems.

The plan is to put it in `MPVRenderer`, along with all the render related code currently in `MPVController`, similar to how mpv structures the code. But I got interrupted doing that refactoring. As we are now looking to freeze changes, test and release the first beta, this refactoring will have to wait for a future release. So this method is here temporarily.